### PR TITLE
Mina Rework and Graphics

### PR DESCRIPTION
**Pull request by orian34**
Will keep updating until I did all the work needed.

Do not merge yet!

### CODE REVIEW

**Comment by orian34** [comment]
Mistake, wanted to assign you only.

File: Update/_mina_001.txt [hunk 1]
```diff
@@ -410,12 +399,8 @@ void main()
 		   NULL, "I can't remember how long I've been living just for the sake of obtaining that simple happiness.", Line_Normal);
 	ClearMessage();
 
-	DrawScene( "Mo1", 0 );
-
-	DisableWindow();
-	DrawBustshot( 7, "filter_Hanyu", 0, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 12, 400, TRUE );
-	DrawBustshotWithFiltering( 0, "Ha_YowakiA1%", "mask00", 1, 0, 0, FALSE, 0, 0, 0, 0, 0, 9*2, 300, TRUE );
-	DisplayWindow();
+	DrawBustshot( 0, "sprite/transparent/ha1_yowaki_", 0, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 5, 0, FALSE );
```
Comment (enumag) [inline Update/_mina_001.txt]: This is a mistake. DrawBustshot is not LS compatible so it needs the full path to the sprite - including the 0/1/2 at the end which is missing here. Grelo usually used 0. The number is not meaningless though - it changes which variant will be shown when LS is disabled.

File: Update/_mina_001.txt [hunk 2]
```diff
@@ -1074,17 +1047,16 @@ void main()
...
 	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#6972c1>梨花</color>", NULL, "<color=#6972c1>Rika</color>", NULL, Line_ContinueAfterTyping); }
 	PlayVoice(4, "ps3/s12/05/180500024", 128);
 	OutputLine(NULL, "「…………………？」",
 		   NULL, "\".........?\"", Line_Normal);
 	ClearMessage();
 
-	DrawBustshotWithFiltering( 2, "Sa_AkireruB1#", "maskright", 1, 160, 0, FALSE, 0, 0, 0, 0, 0, 10, 0, FALSE );
-	DrawSceneWithMask( "Mo2", "maskright", 1, 0, 300 );
+	DrawBustshot( 0, "sprite/normal/sa2b_akireru_b1_", 160, 0, 0, FALSE, 0, 0, 0, 0, 0, 0, 0, 5, 0, FALSE );
+	DrawSceneWithMask( "background/Mo2", "maskleft", 1, 0, 300 );
```
Comment (enumag) [inline Update/_mina_001.txt]: use lowercase paths (everywhere) - it's necessary for linux compatibility

```suggestion
	DrawSceneWithMask( "background/mo2", "maskleft", 1, 0, 300 );
```

File: Update/_mina_001.txt (hunk 1 above)
Comment (orian34) [reply Update/_mina_001.txt]: gotcha, fixed it all

File: Update/_mina_001.txt (hunk 2 above)
Comment (orian34) [reply Update/_mina_001.txt]: was unsure, changed all to lowercase now 👌

**Pull request merged**